modules/sandboxed-containers-prereqs-peer-pods.adoc

```diff
@@ -6,92 +6,38 @@
 [id="sandboxed-containers-prerequisites-peer-pods_{context}"]
 = Prerequisites
 
-Before you install {sandboxed-containers-first}, ensure that your {product-title} cluster meets the following requirements:
+Before you install {sandboxed-containers-first} and enable peer-pods, you must meet the following requirements:
 
-* Your cluster must be installed on on-premise bare-metal infrastructure with {op-system-first} workers. You can use any installation method including user-provisioned, installer-provisioned, or assisted installer to deploy your cluster.
-+
-[NOTE]
-====
-* {sandboxed-containers-first} only supports {op-system} worker nodes. {op-system-base} nodes are not supported.
-* Nested virtualization is not supported.
-====
-
-* You can install {sandboxed-containers-first} on Amazon Web Services (AWS) bare-metal instances. Bare-metal instances offered by other cloud providers are not supported.
-
-[id="sandboxed-containers-resource-requirements_{context}"]
-== Resource requirements for {sandboxed-containers-first}
-
-{sandboxed-containers-first} lets users run workloads on their {product-title} clusters inside a sandboxed runtime (Kata). Each pod is represented by a virtual machine (VM). Each VM runs in a QEMU process and hosts a `kata-agent` process that acts as a supervisor for managing container workloads, and the processes running in those containers. Two additional processes add more overhead:
-
-* `containerd-shim-kata-v2` is used to communicate with the pod.
-* `virtiofsd` handles host file system access on behalf of the guest.
-
-Each VM is configured with a default amount of memory. Additional memory is hot-plugged into the VM for containers that explicitly request memory.
-
-A container running without a memory resource consumes free memory until the total memory used by the VM reaches the default allocation. The guest and its I/O buffers also consume memory.
-
-If a container is given a specific amount of memory, then that memory is hot-plugged into the VM before the container starts.
+* You have {product-title} {product-version} installed on AWS or Azure.
```

Comment:
Since vSphere is already mentioned, should the above line also include vSphere ?
cc @@jensfr

Reply:
@bpradipt  Sorry, the section about vSphere should be removed. Not sure why it's still here.